### DISCUSSION TITLE
Is there any data collected by The Composable architecture?

### DISCUSSION THREAD

**Discussion by stremsdoerfer** [pointfreeco/swift-composable-architecture#1039] [Q&A] [answered]
Hi !

In order for us to provide accurate Privacy Nutrition Labels to Apple during our review, we need to know whether all our third party libraries collect any kind of data. Is there any data collected by the composable architecture that we should specify in our review? (such as crash, usage, diagnostics etc.)

Thanks!

**Comment by mbrandonw** [comment, marked as answer]
Nope, no data is collected or transmitted to a 3rd party.

**Reply by stremsdoerfer** [reply]
Awesome, thanks for your quick response.
